pkg/kubectl/autoscale.go

```diff
@@ -38,6 +39,7 @@ func (HorizontalPodAutoscalerV1Beta1) ParamNames() []GeneratorParam {
 		{"scaleRef-kind", false},
 		{"scaleRef-name", false},
 		{"scaleRef-apiVersion", false},
+		{"scaleRef-apiGroup", false},
```

Comment:
Please consider making a single `scaleRef-apiGroupVersion` instead of having two separate things. It makes little sense to have a version without a group.